dbms/src/Flash/Planner/plans/PhysicalSource.cpp

```diff
@@ -1,38 +0,0 @@
-// Copyright 2022 PingCAP, Ltd.
```

Comment:
PhysicalSource only use for DAGQueryBlock.
After DAGQueryBlock removed, it can be removed.